data/sample.lua

```diff
@@ -143,12 +143,12 @@ end
 
 function Link(s, src, tit, attr)
   return "<a href='" .. escape(src,true) .. "' title='" ..
-         escape(tit,true) .. "'>" .. s .. "</a>"
+          escape(tit,true) .. "'>" .. s .. "</a>"
```

Comment:
Yes it fails because in this case the setting is 2 spaces, so all indentation needs to be a multiple of 2 (i.e. odd spacing).

All the (cosmetic) changes I've done to the sources were due to validations errors.